zetaclient/core_context/zeta_core_context_test.go

```diff
@@ -0,0 +1,261 @@
+package corecontext
```

Comment:
```suggestion
package corecontext_test
```
Ideally it is better to use test package unless there are methods we need to keep private